style.css

```diff
@@ -0,0 +1,45 @@
+#main-doc {
+  display: flex;
+  flex-direction: row-reverse;
+  /* position: absolute; */
+}
+maain {
```

Comment:
Kindly use the correct names when selecting elements. This will greatly contribute to the efficiency of your CSS code during styling.